src/Acuminator/Acuminator.Analyzers/StaticAnalysis/EventHandlers/EventHandlerAnalyzer.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Acuminator.Analyzers.StaticAnalysis.ConnectionScopeInRowSelecting;
+using Acuminator.Utilities.Roslyn;
+using Acuminator.Utilities.Roslyn.Semantic;
+using Acuminator.Utilities.Roslyn.Syntax;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Diagnostics;
+
+namespace Acuminator.Analyzers.StaticAnalysis.EventHandlers
+{
+	[DiagnosticAnalyzer(LanguageNames.CSharp)]
+	public class EventHandlerAnalyzer : PXDiagnosticAnalyzer
+	{
+		private readonly ImmutableArray<IEventHandlerAnalyzer> _innerAnalyzers;
+
+		public override ImmutableArray<DiagnosticDescriptor> SupportedDiagnostics { get; }
+
+		public EventHandlerAnalyzer() : this(
```

Comment:
Do we need this constructor? It looks like it can be removed.

Reply:
It is used by Roslyn engine to create a diagnostic.